django/core/checks/model_checks.py

```diff
@@ -25,6 +29,17 @@ def check_all_models(app_configs=None, **kwargs):
             )
         else:
             errors.extend(model.check(**kwargs))
+    for db_table, model_labels in db_table_attr_map.items():
+        if len(model_labels) <= 1:
```

Comment:
It cannot possibly be `< 1`, else the key wouldn't be in the dict. Using `<= 1` instead of `== 1` only makes it more confusing IMHO.

Reply:
Yeah, makes sense. I've also added a comment with this check just for clarity.